test/unit/query/builder.js

```diff
@@ -2498,6 +2498,31 @@ describe("QueryBuilder", function() {
     });
   });
 
+  it("count distinct with multiple columns", function() {
```

Comment:
+1 for writing test first <3 Also integration smoke test would be needed though to make sure that syntax is correct.